### PR TITLE
60FPS: Fix WORLDMAP wait frames in the script + Small fix of FF8

### PR DESCRIPTION
**Pull request by tangtang95**

### CODE REVIEW

File: src/movies.cpp [hunk 1]
```diff
@@ -349,6 +349,8 @@ void ff8_prepare_movie(uint32_t disc, uint32_t movie)
 			if(res > 0) camOffset += res;
 		}
 
+		fclose(camFile);
```
Comment (julianxhokaxhiu) [inline src/movies.cpp:352]: Shouldn't this be inside the `if`? If you're on disk 4 this code will crash as it is.

**Review by julianxhokaxhiu — CHANGES_REQUESTED** [review]

File: src/movies.cpp (hunk 1 above)
Comment (tangtang95) [reply src/movies.cpp:352]: Which if? Right now I'm closing it after it has been read inside the while

Comment (julianxhokaxhiu) [reply src/movies.cpp:352]: The if block just above this line. Where the `fopen` happens :)

**Review by julianxhokaxhiu — CHANGES_REQUESTED** [review]

File: src/movies.cpp (hunk 1 above)
Comment (tangtang95) [reply src/movies.cpp:352]: If you are talking about the if block on line 339, that enters only if the file is not opened correctly. I don't think there is the need to close it when it is not opened

Comment (julianxhokaxhiu) [reply src/movies.cpp:352]: Can you just drop this commit for now and we can re-evaluate later? Thanks!

Comment (tangtang95) [reply src/movies.cpp]: Done

**Review by julianxhokaxhiu — APPROVED** [review]

**Comment by julianxhokaxhiu** [comment]
Thank you!

**Pull request merged**